### PR TITLE
MVP-1078

### PR DESCRIPTION
**Pull request by arapoff**
- modified everything taking into account your certain suggestions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,35 +1,53 @@
-## This is a basic script to convert .csv Allen Brain Atlas datasets into .h5ad AnnData format
-
-The script was created following the [Cell Types Database: RNA-Seq Data](https://portal.brain-map.org/atlases-and-data/rnaseq) and using [this](https://knowledge.brain-map.org/data/CCDLINBDBP7KYYBOXOJ/summary) example from the Allen Brain Map data portal ([NeMO archive here](http://data.nemoarchive.org/biccn/grant/u19_zeng/zeng/transcriptome/scell/SSv4/mouse/processed/analysis/SMARTer_cells_MOp/))
```
Comment (evanbiederstedt) [inline README.md:3]: We should add some background information here I think

**Review by evanbiederstedt — COMMENTED** [review]
I should let @mfutey do the review here first, given it was based on her feedback

**Review by mfutey — APPROVED** [review]
The updates to the readme are a slimmed-down version of my suggestions; which I think are sufficient to guide a biologist user. I had included information about  how to change directories and create file paths that may not be necessary depending on his experience level.

**Review by evanbiederstedt — APPROVED** [review]
Great

Let's share with Jeremy and then revisit if there are any questions

**Pull request merged**